WhaleSpotting/Models/Response/WhaleSightingResponse.cs

```diff
@@ -0,0 +1,23 @@
+using WhaleSpotting.Models.Database;
+
+namespace WhaleSpotting.Models.Response;
+
+public class WhaleSightingResponse
+{
+    private readonly WhaleSighting _whaleSighting;
+
+    public WhaleSightingResponse(WhaleSighting whaleSighting)
+    {
+        _whaleSighting = whaleSighting;
+    }
+    public int Id => _whaleSighting.Id;
+    public DateTime DateOfSighting => _whaleSighting.DateOfSighting;
+    public float LocationLatitude => _whaleSighting.LocationLatitude;
+    public float LocationLongitude => _whaleSighting.LocationLongitude;
+    public string PhotoImageURL => _whaleSighting.PhotoImageURL;
+    public string Description => _whaleSighting.Description;
+    public int NumberOfWhales => _whaleSighting.NumberOfWhales;
+    public ApprovalStatus ApprovalStatus => _whaleSighting.ApprovalStatus;
+    public WhaleSpeciesResponse WhaleSpecies => new WhaleSpeciesResponse(_whaleSighting.WhaleSpecies) ;
+    public UserResponse User => new UserResponse(_whaleSighting.User);
```

Comment:
Instead of using this syntax can we write the Response Models like this
```suggestion
    public int Id { get; set; }
    public DateTime DateOfSighting { get; set; }
    public float LocationLatitude { get; set; }
    public float LocationLongitude { get; set; }
    public string PhotoImageURL { get; set; }
    public string Description { get; set; }
    public int NumberOfWhales { get; set; }
    public ApprovalStatus ApprovalStatus { get; set; }
    public WhaleSpeciesResponse WhaleSpecies { get; set; }
    public UserResponse User { get; set; }
    
    public WhaleSightingResponse(WhaleSighting whaleSighting)
    {
        Id = whaleSighting.Id;
        DateOfSighting = whaleSighting.DateOfSighting;
        LocationLatitude = whaleSighting.LocationLatitude;
        LocationLongitude= whaleSighting.LocationLongitude;
        PhotoImageURL = whaleSighting.PhotoImageURL;
        Description = whaleSighting.Description;
        NumberOfWhales = whaleSighting.NumberOfWhales;
        ApprovalStatus = whaleSighting.ApprovalStatus;
        WhaleSpecies = new WhaleSpeciesResponse(whaleSighting.WhaleSpecies) ;
        User = new UserResponse(whaleSighting.User);
    }
```